### ISSUE TITLE
Beta release does not compile [Windows 10]

### ISSUE THREAD

**Issue by Songtronix** [hecrj/iced#120]
the crates.io version ``0.1.0-beta`` does not compile:
```rust
error[E0599]: no method named `expect` found for type `std::result::Result<(), winit::event_loop::EventLoopClosed<Message>>` in the current scope
   --> C:\Users\songtronix\.cargo\registry\src\github.com-1ecc6299db9ec823\iced_winit-0.1.0-alpha\src\application.rs:377:18
    |
377 |                 .expect("Send command result to event loop");
    |                  ^^^^^^ method not found in `std::result::Result<(), winit::event_loop::EventLoopClosed<Message>>`    |
    = note: the method `expect` exists but the following trait bounds were not satisfied:
            `winit::event_loop::EventLoopClosed<Message> : std::fmt::Debug`

error: aborting due to previous error
```

However current master does compile.

**Comment by Friz64** [comment]
I can reproduce this on Linux with `rustc 1.40.0-beta.5`. The problem is within the `iced_winit` crate.

**Comment by hecrj** [comment]
Oh, thanks for the catch!

The new alpha release of `winit` has breaking changes. I fixed this in #121.
